Remotion/ObjectBinding/Web.Development.WebTesting.TestSite/Controls/BocListUserControl.ascx

```diff
@@ -370,4 +370,23 @@
     </td>
     <td>&nbsp; (always invalid)</td>
   </tr>
+  <tr>
+    <td></td>
+    <td>
+      <testsite:TestBocListWithRowMenuItems

```

Comment:
Do we always use this type  in tests? Otherwise, I'd recommend using a regular boclist not the customized version for this test to simplify the setup / documentation of intent.

Reply:
This type is always used for those tests.